src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TFunction } from 'i18next'
+import { useGetConsensusValidators, Validator } from '../../../oasis-nexus/api'
+import { SearchScope } from '../../../types/searchScope'
+import { SnapshotCard } from '../../components/Snapshots/SnapshotCard'
+import { PieChart } from '../../components/charts/PieChart'
+import { API_MAX_TOTAL_COUNT } from '../../config'
+
+type ValidatorsStats = {
+  label: string
+  value: number
+}
+
+// TODO: remove if backend will provide validators stats
+function countValidatorsState(t: TFunction, validators: Validator[] | undefined): ValidatorsStats[] {
+  let activeCount = 0
+  let inactiveCount = 0
+  if (!validators) return []
+
+  validators.forEach(validator => {
+    if (validator.active) {
+      activeCount++
+    } else {
+      inactiveCount++
+    }
+  })
+
+  return [
+    { label: t('validator.active'), value: activeCount },
+    { label: t('validator.inactive'), value: inactiveCount },
+  ]
```

Comment:
Note: we'll probably want more options https://github.com/oasisprotocol/oasis-wallet-web/issues/748